event-log/README.md

```diff
@@ -0,0 +1,243 @@
+# event-log
+
+This is a microservice which provides CRUD operations for Event Log DB.
+
+## API
+
+| Method | Path                                     | Description                                                    |
+|--------|------------------------------------------|----------------------------------------------------------------|
+|  POST  | ```/events```                            | Creates an event with a `NEW` status                           |
+|  PATCH | ```/events```                            | Changes events' data by applying the given patch               |
+|  GET   | ```/events/latest```                     | Finds events for all the projects with the latest `event_date` |
```

Comment:
How about `GET /events?order_by=event_date`

Reply:
Actually, [our own docs](https://renku.readthedocs.io/en/latest/developer/contributing/api-guidelines.html#sorting) suggest `GET /events?sort=event_date:ASC`

Reply:
I quite like the idea of sorting. The only thing not that obvious to me is how to make the resource to somehow explain that the list you get contains one event (with the most recent date) per project. In other words, you get a list of the `project -> latest event` tuples. Any idea?

Reply:
Then I would just go for `GET /events?latest_per_project=true`